src/core/Form/TextInput/TextInput.baseStyles.tsx

```diff
@@ -12,22 +12,38 @@ export const baseStyles = withSuomifiTheme(
     ${inputContainer({ theme })}
   }
 
+  & .fi-text-input_statusText {
+    display: flex;
+    flex-direction: column;
+
+    & .fi-text-input_statusText_label {
+      ${theme.typography.bodySemiBoldSmall}
+    }
+  }
+
   & .fi-text-input_input {
     ${input({ theme })}
     background-color: ${theme.colors.whiteBase};
   }
 
   &.fi-text-input--error {
     & .fi-text-input_input {
-      color: ${theme.colors.alertBase};
       border-color: ${theme.colors.alertBase};
     }
+    & .fi-text-input_statusText_label {
+      color: ${theme.colors.alertBase};
+    }
   }
   &.fi-text-input--success {
     & .fi-text-input_input {
-      color: ${theme.colors.successBase};
       border-color: ${theme.colors.successBase};
     }
   }
+  &.fi-text-input--disabled {
+    & .fi-text-input_input {
+      color: ${theme.colors.depthBase};
+      background-color: ${theme.colors.depthLight30};
```

Comment:
is this up to date? Was the tokens update already merged?

Reply:
No, not yet rebased to recent changes on develop. Will have to do it next.